frontend/src/scenes/error-tracking/ErrorTrackingGroupScene.tsx

```diff
@@ -3,11 +3,10 @@ import './ErrorTracking.scss'
 import { LemonButton, LemonDivider } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
 import { PageHeader } from 'lib/components/PageHeader'
-import { base64Decode } from 'lib/utils'
 import { useEffect } from 'react'
 import { SceneExport } from 'scenes/sceneTypes'
 
-import { ErrorTrackingGroup } from '~/queries/schema'
+import { ErrorTrackingIssue } from '~/queries/schema'
```

Comment:
WHAAAT is a `~` import 🫨

Reply:
You can do a module import in JS which goes from the root rather than using relative paths. Typically it's denoted with an `@` symbol but I guess we've changed ours

Reply:
Huh, neat